Source/SwiftLintCore/Extensions/SwiftSyntax+SwiftLint.swift

```diff
@@ -184,6 +184,25 @@ public extension EnumDeclSyntax {
             return rawValueTypes.contains(identifier)
         }
     }
+
+    /// True if this enum is a `CodingKey`
+    var isCodingKey: Bool {
```

Comment:
The enum itself is not a coding key, but it defines the coding keys, doesn't it? So what about the name `definesCodingKeys` instead?